src/Compilers/CSharp/Portable/Lowering/Instrumentation/DebugInfoInjector_SequencePoints.cs

```diff
@@ -170,14 +160,19 @@ internal static void GetBreakpointSpan(VariableDeclaratorSyntax declaratorSyntax
 
         internal static void GetFirstLocalOrFieldBreakpointSpan(SyntaxToken? firstToken, VariableDeclaratorSyntax declaratorSyntax, out SyntaxNode node, out TextSpan? part)
         {
+            Debug.Assert(declaratorSyntax.Parent != null);
+
             var declarationSyntax = (VariableDeclarationSyntax)declaratorSyntax.Parent;
 
+            Debug.Assert(declarationSyntax.Parent != null);
```

Comment:
I can't leave the comment on the proper line, but there's an odd empty line added below, at line 175.